From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: root <root@ROBERT-PC.localdomain>
Date: Sat, 11 Jun 2022 10:01:06 +0200
Subject: [PATCH] GERMAN BUKKITMAIN


diff --git a/src/main/java/org/bukkit/craftbukkit/Main.java b/src/main/java/org/bukkit/craftbukkit/Main.java
index 3080b2a7920a736880c48b66164ed45686ab1d44..27b6cafad79ebde7585f9226c225519ee6ef7a3d 100644
--- a/src/main/java/org/bukkit/craftbukkit/Main.java
+++ b/src/main/java/org/bukkit/craftbukkit/Main.java
@@ -182,7 +182,7 @@ public class Main {
                 acceptsAll(asList("server-name"), "Name of the server")
                         .withRequiredArg()
                         .ofType(String.class)
-                        .defaultsTo("Unknown Server")
+                        .defaultsTo("Unbekannt") // 0xBUKKIT
                         .describedAs("Name");
                 // Paper end
             }
@@ -244,7 +244,7 @@ public class Main {
 
             float javaVersion = Float.parseFloat(System.getProperty("java.class.version"));
             if (javaVersion < 61.0) {
-                System.err.println("Unsupported Java detected (" + javaVersion + "). This version of Minecraft requires at least Java 17. Check your Java version with the command 'java -version'.");
+                System.err.println("DU BRAUCHST JAVA 17+ UM DEN SERVER ZU STARTEN."); // 0xBUKKIT
                 return;
             }
 
@@ -310,7 +310,7 @@ public class Main {
                 }
                 // Paper end
                 System.setProperty( "library.jansi.version", "Paper" ); // Paper - set meaningless jansi version to prevent git builds from crashing on Windows
-                System.out.println("Loading libraries, please wait...");
+                System.out.println("[0xSERVER] LADE DEN SERVER, BITTE WARTEN"); // 0xBUKKIT
                 net.minecraft.server.Main.main(options);
             } catch (Throwable t) {
                 t.printStackTrace();
