From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: root <root@ROBERT-PC.localdomain>
Date: Sat, 11 Jun 2022 10:20:24 +0200
Subject: [PATCH] GERMAN PUFFERFISHCONFIG

puf

puf

B10 - TURN PETAL PATCHES ON BY DEFAULT

diff --git a/src/main/java/gg/pufferfish/pufferfish/PufferfishConfig.java b/src/main/java/gg/pufferfish/pufferfish/PufferfishConfig.java
index 95995c7f7e035bac7a4f429d84bb44fec41e01f9..6289b8424faac0a44531b4ecf4523c7441b742ff 100644
--- a/src/main/java/gg/pufferfish/pufferfish/PufferfishConfig.java
+++ b/src/main/java/gg/pufferfish/pufferfish/PufferfishConfig.java
@@ -70,10 +70,9 @@ public class PufferfishConfig {
 		
 		getString("info.version", "1.0");
 		setComment("info",
-				"Pufferfish Configuration",
-				"Check out Pufferfish Host for maximum performance server hosting: https://pufferfish.host",
-				"Join our Discord for support: https://discord.gg/reZw4vQV9H",
-				"Download new builds at https://ci.pufferfish.host/job/Pufferfish");
+				"PUFFERFISH KONFIGURATION", // 0xBUKKIT
+				"https://discord.gg/reZw4vQV9H", // 0xBUKKIT
+				""); // 0xBUKKIT
 		
 		for (Method method : PufferfishConfig.class.getDeclaredMethods()) {
 			if (Modifier.isStatic(method.getModifiers()) && Modifier.isPrivate(method.getModifiers()) && method.getParameterCount() == 0 &&
@@ -232,7 +231,7 @@ public class PufferfishConfig {
 	public static boolean enableAsyncEntityTracker;
 	public static boolean enableAsyncEntityTrackerInitialized;
 	private static void asyncEntityTracker() {
-		boolean temp = getBoolean("enable-async-entity-tracker", false,
+		boolean temp = getBoolean("enable-async-entity-tracker", true,
 				"Whether or not async entity tracking should be enabled.");
 		if (!enableAsyncEntityTrackerInitialized) {
 			enableAsyncEntityTrackerInitialized = true;
@@ -243,7 +242,7 @@ public class PufferfishConfig {
 	public static boolean enableAsyncPathfinding;
 	public static boolean enableAsyncPathfindingInitialized;
 	private static void asyncPathfinding() {
-		boolean temp = getBoolean("enable-async-pathfinding", false,
+		boolean temp = getBoolean("enable-async-pathfinding", true,
 				"Whether or not async pathfinding should be enabled.");
 		if (!enableAsyncPathfindingInitialized) {
 			enableAsyncPathfindingInitialized = true;
