From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Robert Jansen <63073556+rotvproHD@users.noreply.github.com>
Date: Mon, 8 Aug 2022 16:38:30 +0200
Subject: [PATCH] GERMAN MINECRAFTSERVER


diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
index 60bed3f87016b29508ea1a94db65a1f60d960de1..9edf321a29403aaaddfcb85e5943baf2826a4489 100644
--- a/src/main/java/net/minecraft/server/MinecraftServer.java
+++ b/src/main/java/net/minecraft/server/MinecraftServer.java
@@ -921,7 +921,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
             if (this.hasStopped) return;
             this.hasStopped = true;
         }
-        if (!hasLoggedStop && isDebugging()) io.papermc.paper.util.TraceUtil.dumpTraceForThread("Server stopped"); // Paper
+        if (!hasLoggedStop && isDebugging()) io.papermc.paper.util.TraceUtil.dumpTraceForThread("Server gestoppt"); // Paper // 0xBUKKIT
         // Paper start - kill main thread, and kill it hard
         shutdownThread = Thread.currentThread();
         org.spigotmc.WatchdogThread.doStop(); // Paper
@@ -940,8 +940,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
             this.cancelRecordingMetrics();
         }*/ // Purpur
 
-        MinecraftServer.LOGGER.info("Stopping server");
-        Commands.COMMAND_SENDING_POOL.shutdownNow(); // Paper - Shutdown and don't bother finishing
+        MinecraftServer.LOGGER.info("Stoppe Server"); // 0xBUKKIT
         MinecraftTimings.stopServer(); // Paper
         // Purpur start
         if (upnp) {
@@ -963,13 +962,13 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
 
         this.isSaving = true;
         if (this.playerList != null) {
-            MinecraftServer.LOGGER.info("Saving players");
+            MinecraftServer.LOGGER.info("Speichere Spieler"); // 0xBUKKIT
             this.playerList.saveAll();
             this.playerList.removeAll(this.isRestarting); // Paper
             try { Thread.sleep(100); } catch (InterruptedException ex) {} // CraftBukkit - SPIGOT-625 - give server at least a chance to send packets
         }
 
-        MinecraftServer.LOGGER.info("Saving worlds");
+        MinecraftServer.LOGGER.info("Speichere Welten"); // 0xBUKKIT
         Iterator iterator = this.getAllLevels().iterator();
 
         ServerLevel worldserver;
@@ -1005,7 +1004,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
         io.papermc.paper.chunk.system.io.RegionFileIOThread.close(true); // Paper // Paper - rewrite chunk system
         LOGGER.info("Closing Thread Pool");
         Util.shutdownExecutors(); // Paper
-        LOGGER.info("Closing Server");
+        LOGGER.info("[0xSERVER] Beende Server"); // 0xBUKKIT
         try {
             net.minecrell.terminalconsole.TerminalConsoleAppender.close(); // Paper - Use TerminalConsoleAppender
         } catch (Exception e) {
@@ -1036,7 +1035,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
         org.purpurmc.purpur.task.BeehiveTask.instance().unregister(); // Purpur
         this.isRestarting = isRestarting;
         this.hasLoggedStop = true; // Paper
-        if (isDebugging()) io.papermc.paper.util.TraceUtil.dumpTraceForThread("Server stopped"); // Paper
+        if (isDebugging()) io.papermc.paper.util.TraceUtil.dumpTraceForThread("[0xSERVER] Server ist offline"); // Paper // 0xBUKKIT
         // Paper end
         this.running = false;
         if (flag) {
@@ -1133,7 +1132,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
             // anything at 3+ won't be caught here but also will trip watchdog....
             // tasks are default scheduled at -1 + delay, and first tick will tick at 1
             String doneTime = String.format(java.util.Locale.ROOT, "%.3fs", (double) (Util.getNanos() - serverStartTime) / 1.0E9D);
-            LOGGER.info("Done ({})! For help, type \"help\"", doneTime);
+            LOGGER.info("Done ({})! Der Server ist nun Online!", doneTime); // 0xBUKKIT
             // Paper end
 
             org.spigotmc.WatchdogThread.tick(); // Paper
@@ -1154,7 +1153,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
                     long j = i / 50L;
 
                     if (this.server.getWarnOnOverload()) // CraftBukkit
-                    MinecraftServer.LOGGER.warn("Can't keep up! Is the server overloaded? Running {}ms or {} ticks behind", i, j);
+                            MinecraftServer.LOGGER.warn("[0xSERVER] Ist der Server überfordert? {}ms oder {} ticks hinterher!", i, j); // 0xBUKKIT
                     this.nextTickTime += j * 50L;
                     this.lastOverloadWarning = this.nextTickTime;
                 }
@@ -1682,11 +1681,11 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
 
     @DontObfuscate
     public String getServerModName() {
-        return org.purpurmc.purpur.PurpurConfig.serverModName; // Purpur - Purpur > // Paper - Paper > // Spigot - Spigot > // CraftBukkit - cb > vanilla!
+        return org.purpurmc.purpur.PurpurConfig.serverModName; // PAPERSTUDIOS - 0xBUKKIT > // Purpur - Purpur > // Paper - Paper > // Spigot - Spigot > // CraftBukkit - cb > vanilla!
     }
 
     public SystemReport fillSystemReport(SystemReport details) {
-        details.setDetail("Server Running", () -> {
+        details.setDetail("Server Online", () -> {
             return Boolean.toString(this.running);
         });
         if (this.playerList != null) {
@@ -1999,7 +1998,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
     @Override
     public void executeIfPossible(Runnable runnable) {
         if (this.isStopped()) {
-            throw new RejectedExecutionException("Server already shutting down");
+            throw new RejectedExecutionException("Server stoppt schon!"); // 0xBUKKIT
         } else {
             super.executeIfPossible(runnable);
         }
