From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: root <root@ROBERT-PC.localdomain>
Date: Tue, 14 Jun 2022 11:18:39 +0200
Subject: [PATCH] GERMAN MINECRAFTMAIN


diff --git a/src/main/java/net/minecraft/server/Main.java b/src/main/java/net/minecraft/server/Main.java
index 0066b1abc008d245825abf1d256cb87fa9c2d877..d22c873c5f863c1b1128fada7092d89b9cb600d2 100644
--- a/src/main/java/net/minecraft/server/Main.java
+++ b/src/main/java/net/minecraft/server/Main.java
@@ -145,13 +145,11 @@ public class Main {
             boolean eulaAgreed = Boolean.getBoolean( "com.mojang.eula.agree" );
             if ( eulaAgreed )
             {
-                System.err.println( "You have used the Spigot command line EULA agreement flag." );
-                System.err.println( "By using this setting you are indicating your agreement to Mojang's EULA (https://account.mojang.com/documents/minecraft_eula)." );
-                System.err.println( "If you do not agree to the above EULA please stop your server and remove this flag immediately." );
+                System.err.println( "EULA automatisch akzeptiert." ); // 0xBUKKIT
             }
             // Spigot End
             if (!eula.hasAgreedToEULA() && !eulaAgreed) { // Spigot
-                Main.LOGGER.info("You need to agree to the EULA in order to run the server. Go to eula.txt for more info.");
+                Main.LOGGER.info("Bitte akzeptiere die EULA in EULA.txt"); // 0xBUKKIT
                 return;
             }
 
@@ -213,7 +211,7 @@ public class Main {
             try {
                 com.google.common.io.Files.write("{\n"
                         + "    \"pack\": {\n"
-                        + "        \"description\": \"Data pack for resources provided by Bukkit plugins\",\n"
+                        + "        \"description\": \"BUKKIT\",\n" // 0xBUKKIT
                         + "        \"pack_format\": " + SharedConstants.getCurrentVersion().getPackVersion() + "\n"
                         + "    }\n"
                         + "}\n", mcMeta, com.google.common.base.Charsets.UTF_8);
