From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: root <root@ROBERT-PC.localdomain>
Date: Mon, 13 Jun 2022 12:21:47 +0200
Subject: [PATCH] GERMAN MINECRAFTPROPERTIES


diff --git a/src/main/java/net/minecraft/server/dedicated/Settings.java b/src/main/java/net/minecraft/server/dedicated/Settings.java
index bafd0f3492a0b8778d28610785129c5eba7fe4a2..b4137f09ea66a7ac2562c68ec88d1e7e8ba70fb0 100644
--- a/src/main/java/net/minecraft/server/dedicated/Settings.java
+++ b/src/main/java/net/minecraft/server/dedicated/Settings.java
@@ -121,7 +121,7 @@ public abstract class Settings<T extends Settings<T>> {
             // Paper end
 
             try {
-                this.properties.store(bufferedOutputStream, "Minecraft server properties"); // Paper - use bufferedOutputStream
+                this.properties.store(bufferedOutputStream, "MINECRAFT KONFIGURATION"); // Paper - use bufferedOutputStream // 0xBUKKIT
             } catch (Throwable throwable) {
                 if (outputstream != null) {
                     try {
