From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Robert Jansen <63073556+rotvproHD@users.noreply.github.com>
Date: Fri, 5 Aug 2022 08:46:05 +0200
Subject: [PATCH] GERMAN PUFFERFISHCONFIG


diff --git a/src/main/java/gg/pufferfish/pufferfish/PufferfishConfig.java b/src/main/java/gg/pufferfish/pufferfish/PufferfishConfig.java
index dddbcfb9230970149c7149567c255dc0b383a660..b0edbf2b8a064da9276d77a998ea349e8ba76d88 100644
--- a/src/main/java/gg/pufferfish/pufferfish/PufferfishConfig.java
+++ b/src/main/java/gg/pufferfish/pufferfish/PufferfishConfig.java
@@ -72,10 +72,8 @@ public class PufferfishConfig {
 		
 		getString("info.version", "1.0");
 		setComment("info",
-				"Pufferfish Configuration",
-				"Check out Pufferfish Host for maximum performance server hosting: https://pufferfish.host",
-				"Join our Discord for support: https://discord.gg/reZw4vQV9H",
-				"Download new builds at https://ci.pufferfish.host/job/Pufferfish");
+				"PUFFERFISH KONFIGURATION",
+				"https://discord.gg/reZw4vQV9H");
 		
 		for (Method method : PufferfishConfig.class.getDeclaredMethods()) {
 			if (Modifier.isStatic(method.getModifiers()) && Modifier.isPrivate(method.getModifiers()) && method.getParameterCount() == 0 &&
