From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: root <root@ROBERT-PC.localdomain>
Date: Mon, 13 Jun 2022 12:21:47 +0200
Subject: [PATCH] GERMAN MINECRAFTPROPERTIES


diff --git a/src/main/java/net/minecraft/server/dedicated/Settings.java b/src/main/java/net/minecraft/server/dedicated/Settings.java
index aafa84578c7fb25feeee043259f9c056929ca008..b68d39ca7231ec3d1bfb5e73201dd1d31c0e584f 100644
--- a/src/main/java/net/minecraft/server/dedicated/Settings.java
+++ b/src/main/java/net/minecraft/server/dedicated/Settings.java
@@ -120,7 +120,7 @@ public abstract class Settings<T extends Settings<T>> {
             // Paper end
 
             try {
-                this.properties.store(bufferedOutputStream, "Minecraft server properties"); // Paper - use bufferedOutputStream
+                this.properties.store(bufferedOutputStream, "MINECRAFT KONFIGURATION"); // Paper - use bufferedOutputStream // 0xBUKKIT
             } catch (Throwable throwable) {
                 if (outputstream != null) {
                     try {
