From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: BillyGalbreath <blake.galbreath@gmail.com>
Date: Sun, 12 Jun 2022 10:57:51 -0500
Subject: [PATCH] SERVER KONFIGURATION


diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index a7e2fab3e174fe2c1d79d904f7eaff88dd0c3c82..8186f3f7a0976a005f1f18d932f0be10ceeb04af 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -1981,6 +1981,13 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         }
         // Purpur end
 
+        // 0xBUKKIT start
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration get0xBUKKITConfig() {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // 0xBUKKIT end
+
         /**
          * Sends the component to the player
          *
