From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: BillyGalbreath <blake.galbreath@gmail.com>
Date: Sun, 12 Jun 2022 10:57:51 -0500
Subject: [PATCH] SERVER KONFIGURATION


diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index 72adbee17a6c5cb7f067fb4875ff39e4da3f8ef1..076a7502a2c350f94d5c8fb353ad05a1ae7ac8d1 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -1909,6 +1909,13 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         }
         // Purpur end
 
+        // 0xBUKKIT start
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration get0xBUKKITConfig() {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // 0xBUKKIT end
+
         /**
          * Sends the component to the player
          *
